R/show_rcode_modal.R

```diff
@@ -33,6 +34,4 @@ show_rcode_modal <- function(title = NULL, rcode, session = getDefaultReactiveDo
     size = "l",
     easyClose = TRUE
   ))
-
-  return(NULL)
```

Comment:
This may make sense, actually. I don't know what `showModal` returns but it seems whatever it is, we don't want it. Either verify or leave the `NULL`, please.

Reply:
That function seems to return `NULL`

![image](https://github.com/insightsengineering/teal/assets/211358/0092b142-b16c-4dcc-97a3-e75434af101a)